feat: Adds Detokenization reference docs and removes encryption docs (#163)
Basis-Theory/docs: source/includes/detokenization/_examples.md

# Examples

Based on your use specific cases, you may prefer to store your sensitive data with Basis Theory using our
[Atomic Tokens](/concepts/what-are-atomic-tokens), one of our pre-defined [Token Types](https://docs.basistheory.com/#token-types),
or even use your own custom data schema within a schemaless [generic token](https://docs.basistheory.com/#token-types-token).
However you choose to tokenize your sensitive data, Basis Theory grants you the flexibility to use this data within
our serverless Reactor platform or the Proxy.

In this section, we will walk through several examples of how you can detokenize data within Reactors or Proxy requests 
regardless of how the data was tokenized.

## Prerequisites

> If Reactors or Proxy are completely new to you, we recommend you first read our [Reactors](/concepts/what-are-reactors) 

Reviewer: Both of these links return 404, these concepts are under https://developers.basistheory.com/
```suggestion
If Reactors or Proxy are completely new to you, we recommend you first read our [Reactors](https://developers.basistheory.com//concepts/what-are-reactors) 
```

      -X "POST" \
      -d '{
            "payment_method": {
              "credit_card": {
                "number": "{{815029c2-29ec-4fc2-8cd4-99feb3ee582c | $.number}}"
                "month": "{{815029c2-29ec-4fc2-8cd4-99feb3ee582c | $.expiration_month}}",
                "year": "{{815029c2-29ec-4fc2-8cd4-99feb3ee582c | $.expiration_year}}",
                "verification_value": "987",
                "full_name": "John Doe"
              },
              "retained": true
            }
          }'
```

> Detokenized Request

```shell--reactor
{
  "args": {
    "card": {
      "number": "4242424242424242",
      "expiration_month": 11,
      "expiration_year": 2025,
      "cvc": "987"
    },
    "card_owner_full_name": "John Doe"
  }, 
  "configuration": {...}
}
```

```shell--proxy
{
  "payment_method": {
    "credit_card": {
      "number": "4242424242424242",
      "month": "11",
      "year": "2025",
      "verification_value": "987",
      "full_name": "John Doe"
    },
    "retained": true
  }
}
```

In this example, we show how you can use an [Atomic Card](https://docs.basistheory.com/#atomic-cards-atomic-card-object) token to create a Spreedly payment method,

> but provide an updated CVC (`987`) that is different from the `cvc` value stored with the token.

Reviewer: ```suggestion
but provide an updated CVC (`987`) that is different from the `cvc` value stored within the token.
```

# Examples

> Based on your use specific cases, you may prefer to store your sensitive data with Basis Theory using our

Reviewer 2: ```suggestion
Based on your use case, you may prefer to store your sensitive data with Basis Theory using our
```
